var/spack/repos/builtin/packages/r-acepack/package.py

```diff
@@ -9,8 +9,8 @@
 class RAcepack(RPackage):
     """ACE and AVAS for Selecting Multiple Regression Transformations."""
 
-    homepage = "https://CRAN.R-project.org/package=acepack"
+    homepage = "https://cloud.R-project.org/package=acepack"
```

Comment:
Switch to lowercase R

Reply:
Okay; my script was just looking for cran/cloud, but I should be able to add this.

Reply:
This should be fixed with the latest commit.